docs/02-concepts/02-math/01-elliptic curves.md

```diff
@@ -0,0 +1,53 @@
+# Elliptic Curves
+
+Elliptic curves are a core foundation of modern cryptography techniques and is used extensively in web browsers, digital security products and blockchain protocols. Here we summarise the elliptic curve knowledge required for implementing verkle tries. If elliptic curves are an unfamiliar topic to you, we would suggest diving into [More Reading](#more-reading)
+
+## Elliptic curve points as a group
+
+Using a [carefully designed addition (+) operation](https://andrea.corbellini.name/2015/05/17/elliptic-curve-cryptography-a-gentle-introduction/#group-law), we can form an [abelian group](./00-group.md/#abelian-group) out of the points of an elliptic curve. 
+
+## Scalar multiplication
+
+We can extend the addition operation for elliptic curve points to define scalar multiplication:
+
+$m*P = \underbrace{P + P + \cdots + P}_{m\ \text{times}}$, where $n$ is an integer > 0 and $P$ is an elliptic curve point.
+
+## Multiscalar multiplication
+
+Multiscalar multiplication is when we 'add together' several scalar multiplication results, e.g.
+
+$m*P + n*Q = \underbrace{P + P + \cdots + P}_{m\ \text{times}} + \underbrace{Q + Q + \cdots + Q}_{n\ \text{times}}$
+
+## Elliptic curves over a finite field
+
+We can restrict an elliptic curve such that the x and y coordinates of each point is an element of [$\mathbb{F}_p$](./00-group.md/#integers-modulo-prime---fp), rather than a real number $\mathbb{R}$. More formally, the continuous elliptic curve defined by
+
+$y^2 = x^3 + ax + b$, where $x, y, a, b \in \mathbb{R}$
+
+is restricted to 
+
+$y^2 = x^3 + ax + b\>(mod\>p)$, where $x, y, a, b \in \mathbb{F}_p$ and $p$ is a prime number
+
+This transforms the graph from a continuous curve to a discrete set of points.
+
+![merkle_00](/img/elliptic_curve_0.png)
+**Figure 1.** Elliptic curve defined by $y^2 = x^3 + 2x + 3$ transformed from being defined over $\mathbb{R}$ to $\mathbb{F}_{29}$
+
+The important mathematical result here is that elliptic curve points over a finite field still form an [abelian group](./00-group.md/#abelian-group).
+
+## Elliptic curve discrete logarithm problem
+
+For the scalar multiplication $Q = n*P$ 
+
+Where $P$ is an elliptic curve point over $\mathbb{F}_p$, $n$ is a scalar $\in \mathbb{F}_p$ and $Q$ is the scalar multiplication result (also an elliptic curve point over $\mathbb{F}_p$ by the group property of [closure](./00-group.md/#group-definition)).
+
+- Given $n$ and $P$, it is 'easy' to find $Q$ => Scalar multiplication, known $O(\log{N})$ algorithm
+- Given $Q$ and $P$, it is 'hard' to find $n$ => Discrete logarithm problem, exponential time algorithms for the general case
```

Comment:
Just as a reference, the ECDLP is harder than the general DLP and factorisation problem which is why Elliptic curves public keys for example are usually smaller than RSA public keys